content/applications/hr/payroll/payslips.rst

```diff
@@ -28,159 +30,178 @@ from this dashboard.
+If a payment needs to be canceled or refunded, click the corresponding :guilabel:`Cancel` or
+:guilabel:`Refund` button, located at the top-left of the screen.
 
 .. important::
    In order for a payslip to be paid, the employee *must* have a bank account entered in their
-   contact information. If there is no bank information, a payslip cannot be paid, and an error will
-   appear when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
-   the :guilabel:`Private Information` tab on the employee's card. Edit the employee card, and add
-   banking information, if it is missing.
+   contact information. If there is no bank information, a payslip cannot be paid, and an error
+   appears when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
+   the :ref:`Private Information <employees/private-info>` tab on the employee's card. Edit the
+   employee card, and add banking information if it is missing.
 
    .. image:: payslips/banking.png
       :align: center
       :alt: Banking information can be entered in an employee's card.
 
+.. _payroll/all-payslips:
+
 All payslips
 ============
 
 To view all payslips regardless of status, go to :menuselection:`Payroll app --> Payslips --> All
-Payslips`. Here, all payslips are organized by batch (in a default list view).
+Payslips`. The :guilabel:`Employee Payslips` page loads, displaying all payslips, organized by batch
+(in a default list view).
+
+Click on the :guilabel:`▶ (right arrow)` next to the individual batch name to view all the payslips
+in that particular batch, along with all the payslip details.
 
-Click on the :guilabel:`▶ (arrow)` next to the individual batch name to view all the payslips in
-that particular batch, along with all the payslip details. The number of payslips in the batch is
-written in parenthesis after the batch name. The :guilabel:`Status` for each individual payslip
-appears on the far-right side, indicating if it is in :guilabel:`Draft Mode`, :guilabel:`Waiting`,
-or if it is :guilabel:`Done`.
+The number of payslips in the batch is written in parenthesis after the batch name. The
+:guilabel:`Status` for each individual payslip appears on the far-right side, indicating if it is in
+:guilabel:`Draft Mode`, :guilabel:`Waiting`, :guilabel:`Done`, or if it is :guilabel:`Paid`.
 
 .. note::
    - :guilabel:`Draft` indicates the payslip is created, and there is still time to make edits,
      since the amounts are not calculated.
-   - `Waiting` indicates the payslip has been calculated and the salary details can be found in the
-     *Salary Computation* tab.
-   - `Done` indicates the payslip is calculated and ready to be paid.
+   - :guilabel:`Waiting` indicates the payslip has been calculated and the salary details can be
+     found in the *Salary Computation* tab.
+   - :guilabel:`Done` indicates the payslip is calculated and ready to be paid.
+   - :guilabel:`Paid` indicates the employee has been paid.
```

Comment:
I think this can work as a list without being an admonition -- what do you think?

also, I'd suggest using colons to define each status.

For example:

'Draft': indicated the payslip is ...